extensions/wikia/DataWarehouse/DataWarehouseEventProducer.class.php

```diff
@@ -375,10 +378,21 @@ public function setCategory() {
 		$this->mParams['categoryId'] = WikiFactory::getCategory( $this->app->wg->CityId );
 	}
 
+	public function setBeacon( $beacon ) {
+		$this->mParams['beacon'] = $beacon;
+	}
+
 	public function sendLog() {
 		wfProfileIn( __METHOD__ );
 		$data = json_encode($this->mParams);
 		$this->getRabbit()->publish( $this->mKey, $data );
+		if ( ( ! Wikia::isDevEnv()) && ( $this->mParams['cityId'] % 100 == 28 ) ) { // enabled on 1% of wikis
```

Comment:
Don't you want to use `<` operator instead and just increase the percentage of wikis as we go?

Reply:
Good idea, I will use `<` indeed.